packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -175,6 +201,7 @@ const props = defineProps({
 
 const { i18n } = composables.useI18n()
 const heightRef = ref<string>(props.height)
+const csvFilename = computed<string>(() => props?.chartTitle ? props.chartTitle : i18n.t('csvExport.defaultFilename'))
```

Comment:
Experimenting with this in Konnect: because we don't set a chart title in reports, this way of deriving `csvFilename` means that reports have an empty filename prefix.  I think we need a new optional property on AnalyticsChart so that we can properly set the export filename -- e.g., first try `props.exportFileNamePrefix`, then try `props.chartTitle`, then fall back to a default.

Reply:
Good call. Pushed a commit.